mw/gpio_server/BUILD

```diff
@@ -7,11 +7,11 @@ cc_library(
         "gpio_mw.hpp"
     ],
     deps = [
-        "//core/gpio:gpio_drivers",
+        "//core/gpio:sim_gpio_drivers",
```

Comment:
to nie ma sensu

Reply:
to usunoem juz, zapomniaem podmienic spowrotem